base/broadcast.jl

```diff
@@ -110,6 +129,9 @@ const bitcache_size = 64 * bitcache_chunks # do not change this
 dumpbitcache(Bc::Vector{UInt64}, bind::Int, C::Vector{Bool}) =
     Base.copy_to_bitarray_chunks!(Bc, ((bind - 1) << 6) + 1, C, 1, min(bitcache_size, (length(Bc)-bind+1) << 6))
 
+@inline _broadcast_getindex(A, I) = A
```

Comment:
I wanted to use `broadcast_getindex`, but there is already a function named like that, see https://github.com/JuliaLang/julia/pull/16986/files#diff-8e941cd1878722cc12a54e01537b8e18R274.

It doesn't seem to be used anywhere in base and I don't know if packages rely on it. If we were to change, it would need a change in the documentation and a deprecation.